test_pull_requests

```diff
@@ -485,30 +489,33 @@ class InvalidatingGitHubCache < Faraday::HttpCache
     #
     #  DELETE https://api.github.com/repos/user/repo/issues/comments/111111111
     #    should invalidate: https://api.github.com/repos/user/repo/issues/1/comments?per_page=100
-    #
-    # When called with no arguments, this will invalidate all the
-    # entries in the cache, for when you're about to flush the cache
-    # to disk.
-    def invalidate_cache_entries_matching(url=nil)
+    def invalidate_cache_entries_matching(url)
       if !url.respond_to? :gsub!
         if url.respond_to? :to_s
           url = url.to_s
         else
-          url = nil
+          raise TypeError.new("#{__method__} requires a string or a stringifiable object")
```

Comment:
Still amazed we're somehow managing to pass in something that doesn't match this but ok